Pod/Classes/VOKCoreDataManager.m

```diff
@@ -18,6 +18,8 @@ @interface VOKCoreDataManager ()
 @property (nonatomic, strong) NSManagedObjectModel *managedObjectModel;
 @property (nonatomic, strong) NSPersistentStoreCoordinator *persistentStoreCoordinator;
 @property (nonatomic, strong) NSManagedObjectContext *managedObjectContext;
+///A private parent context for writing to the persistent store
+@property (nonatomic, strong) NSManagedObjectContext *rootContext;
```

Comment:
just my own feeling, but renaming this to something like `privateRootContext` would make this more self-documenting